_rules/disable-shortcut-670a30.md

```diff
@@ -0,0 +1,273 @@
+---
+id: 670a30
+name: Printable keys only shortcut can be disabled
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, a mechanism to disable the shortcut exists.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys.
+
+## Expectation
+
+For each [user interface component][] that is a [descendent][] of the root node of the test target, if a [printable character][] [shortcut][keyboard shortcut] exists to trigger the action of that [user interface component][], it can be disabled.
+
+The [user interface component][] used to disable the [shortcut][keyboard shortcut] must be [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty (`""`).
+
+## Assumptions
+
+This rule assumes as applicable [keyboard shortcuts][keyboard shortcut] those implemented by the test target [content][]. Any other means (e.g. browser extensions, browser settings, user agents, external browser applications) are not considered.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.1.4: Character Key Shortcuts](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html)
+- [G127 Providing a mechanism to allow users to remap or turn off character key shortcuts](https://www.w3.org/WAI/WCAG21/Techniques/general/G217)
+- [F99 Failure of Success Criterion 2.1.4 due to implementing character key shortcuts that cannot be turned off or remapped](https://www.w3.org/WAI/WCAG21/Techniques/failures/F99)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The [HTML document][] has a [keyboard shortcut][] using only a [printable character][], and it can be disabled.
```

Comment:
```suggestion
The [HTML document][] has a [keyboard shortcut][] using only a [printable character][], and it can be disabled by a [user interface component][] which is [visible][], [included in the accessibility tree][], and has a non-empty [accessible name][].
```